### ISSUE TITLE
My account reverted back and most of my tasks were deleted

### ISSUE THREAD

**Issue by codaa27** [HabitRPG/habitica#1212]
Hi,

My user ID is e21df476-9e5b-4c9b-b41d-752c706a9ec0

During the most recent server reboot occurring at around 4:15 PM (some 25 minutes ago) my character got squashed some. He went back down from Level 4  to Level 3 at 129/170 XP, losing some 60 XP in the process, and all of my Habits, Dailies, and To-Dos were wiped except for four tasks.

Unfortunately I made some twenty items in my To-Do for my studying homework and I'm sad to see them all gone. The good news was that using HabitRPG I managed to get 17 hours of work done in the past two days alone. If you can restore my account that would be super.

Much thanks,
David

**Comment by lefnire** [comment]
Unfortunately lost data means the saved content never went through to the
database, since the serve was down or the connection was broken - so
there's nothing to restore to :( Is the data you lost data you added that
day, or stuff that's been around for a while and showing up on refreshes?
If 1, alas :( If 2 I can restore you if you know when was a functional
state for you.

Obviously this is unacceptable app behavior, it's due to bugs in the
underlying framework we're using. I'm currently working very hard on
upgrading the framework and am just about there

**Comment by codaa27** [comment]
I'M A DUM DUM I HAD THE TAG ON ERRANDS.

I wrote a very nice e-mail which I scrapped now because I noticed the
discrepancy to realize I somehow entered the tag filter, which I'm glad
for, and must have made up the rest, which I'm a little embarrassed about.
But I still wanted to share this one with you:

_I don't mind inputting the information again, that it's free and
open-source and here speaks a lot on the number of factors for me to use
this in the first place :-) It's just surprising was all._
*
*
_Thanks a lot,_
_David_

Really, thanks.
David
